src/OnboardingSPA/components/CheckboxTemplate/CheckboxItem/index.js

```diff
@@ -0,0 +1,112 @@
+import { useState } from '@wordpress/element';
+import { Icon, help, box } from '@wordpress/icons';
+
+import { CheckboxControl } from '@wordpress/components';
+
+/**
+ * Checkbox Item Component
+ * This returns a Single Element with a toggable description
+ *
+ * @param {string} icon     - The icon name of the Item
+ * @param {string} title    - The Main Title of the Item
+ * @param {string} subtitle - The Sub Title of the Item
+ * @param {string} desc     - The Description of the Item
+ *
+ * @return CheckboxItem
+ */
+
+const CheckboxItem = ( {
```

Comment:
Probably a bad idea to call it `CheckboxItem` and then pass it something that is very specific to site features which is `slug`. This also needs to take a `className` for custom CSS